docker/test/integration/minifi/validators/SingleOrMoreFileOutputValidator.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+
+from .FileOutputValidator import FileOutputValidator
+
+
+class SingleOrMoreFileOutputValidator(FileOutputValidator):
+    """
+    Validates the content of a single file in the given directory.
+    """
```

Comment:
SingleOrMulti or OneOrMore (my preference) would be better naming. Also, the comment doesn't mention the possibility of multiple matches.